engine/src/flutter/lib/web_ui/lib/src/engine/safe_browser_api.dart

```diff
@@ -11,100 +11,41 @@
 @JS()
 library browser_api;
 
-import 'dart:async';
 import 'dart:js_interop';
-import 'dart:js_util' as js_util;
-import 'dart:math' as math;
-import 'dart:typed_data';
+import 'dart:js_interop_unsafe';
 
-import 'package:ui/ui.dart' as ui;
 import 'package:ui/ui_web/src/ui_web.dart' as ui_web;
 
-import 'browser_detection.dart' show WebGLVersion, webGLVersion;
-import 'display.dart';
 import 'dom.dart';
-import 'vector_math.dart';
-
-/// Returns true if [object] has property [name], false otherwise.
-///
-/// This is equivalent to writing `name in object` in plain JavaScript.
-bool hasJsProperty(Object object, String name) {
-  return js_util.hasProperty(object, name);
-}
-
-/// Returns the value of property [name] from a JavaScript [object].
-///
-/// This is equivalent to writing `object.name` in plain JavaScript.
-T getJsProperty<T>(Object object, String name) {
-  return js_util.getProperty<T>(object, name);
-}
-
-const Set<String> _safeJsProperties = <String>{'decoding', '__flutter_state'};
-
-/// Sets the value of property [name] on a JavaScript [object].
-///
-/// This is equivalent to writing `object.name = value` in plain JavaScript.
-T setJsProperty<T>(Object object, String name, T value) {
-  assert(
```

Comment:
I'm not sure how useful this assertion is. Perhaps we should keep the wrapper just for `setProperty` so this check is maintained?

Reply:
This seems...silly. There is no enforcement. It wasn't used consistently.